index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="zxx">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <div class="wrapper">
+      <nav id="navbar">
+        <header>
+          <h1>JS Documentation</h1>
```

Comment:
Please replace the \<h1\> tag with \<a\> tag. Because, this is the logo and it should go to the home page when clicked.